src/libmongoc/doc/mongoc_auto_encryption_opts_destroy.rst

```diff
@@ -22,5 +22,5 @@ Parameters
 
   | :symbol:`mongoc_auto_encryption_opts_new()`
 
-  | The guide for :doc:`Using Client-Side Field Level Encryption <using_client_side_encryption>`
+  | :doc:`In-Use Encryption <in-use-encryption>`
```

Comment:
The explicit link title is the same as the target page title, so it is redundant on each occurrence, and can be ":doc:`in-use-encryption`".

Reply:
Nice. I was not aware of that. Updated.